lib/Conversation/styles.scss

```diff
@@ -216,25 +221,36 @@ $conversation-meta-text-color: $neutral-base !default;
   display: block;
 }
 
-.is-active .conversation__comment {
+.is-active > .conversation__comment-list .conversation__comment {
+  margin-bottom: 0;
+}
+
+.is-active .conversation__comment,
+.show-reply-preview > .conversation__comment-list .conversation__comment {
   position: relative;
   &:before {
     content: ' ';
-    border-left: $conversation-border-width solid $conversation-border-color;
+    border-left: 2px solid $conversation-border-color;
```

Comment:
This is double the previous value should we *2 the var rather than setting a fixed value? Or just alter the var to be `2px`?

Reply:
Its because that var also controls the outer border of the conversation, this is just the little thread between the two. Before they could share the same value, but some feedback was to change it to 2px, obv I dont want to make all the borders 2px, just this guy.

Reply:
> Its because that var also controls the outer border of the conversation, this is just the little thread between the two. Before they could share the same value, but some feedback was to change it to 2px, obv I dont want to make all the borders 2px, just this guy.

(I can 2* it, but it just seems excessive to me, esp since they may want to be handled differently)